examples/gnrc_border_router/README.md

```diff
@@ -1,4 +1,144 @@
-# gnrc_networking_border_router example
+# gnrc_border_router using automatic configuration
+A new setup is possible on which only one serial interface is needed. 
+This is done through the ethos (Ethernet Over Serial) driver and UHCP.
+Ethos multiplexes serial data to separate ethernet packets from shell commands.
+UHCP is in charge of configure the wireless interface prefix and routes on the BR.
```

Comment:
You should describe what UHCP means here (you did that a few lines below). Also `configure` => `configuring`